website/src/word-panel.css.tsx

```diff
@@ -0,0 +1,39 @@
+import { style } from "@vanilla-extract/css"
+import sprinkles, { hspace, mediaQueries, theme, vspace } from "./sprinkles.css"
+
+export const wholePanel = style({
```

Comment:
I think I mentioned this before, but we do want the panel to fill to the bottom of the screen. This way, it doesn't change height if a word has more information than another.